maven-osgi-package-plugin/src/main/java/at/bestsolution/maven/osgi/pack/ProductPackagePlugin.java

```diff
@@ -67,16 +67,19 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
         Xpp3Dom launcherArgs = new Xpp3Dom("launcherArgs");
         Xpp3Dom programArgs = new Xpp3Dom("programArgs");
+        StringBuilder programArgsVal = new StringBuilder();
         for (String a : product.launcherArgs.programArguments) {
```

Comment:
You could use here `Stream.of(product.launcherArgs.programArguments).collect(Collectors.joining( " " ));` instead of the for loop

Reply:
Done